packages/server/src/rateLimit/index.ts

```diff
@@ -42,7 +42,7 @@ export default class RateLimit {
     });
   }
 
-  shouldRateLimit(ip: string, methodName: string, total: number): boolean {
+  shouldRateLimit(ip: string, requestId: string, methodName: string, total: number): boolean {
```

Comment:
nit: always best to add new params at the end of a method.

Reply:
Agreed.  Updated.